DFC.ServiceTaxonomy.Neo4j/Commands/DeleteNodesByTypeCommand.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DFC.ServiceTaxonomy.Neo4j.Commands.Interfaces;
+using Neo4j.Driver;
+
+namespace DFC.ServiceTaxonomy.Neo4j.Commands
+{
+    public class DeleteNodesByTypeCommand : IDeleteNodesByTypeCommand
+    {
+        public HashSet<string> NodeLabels { get; set; } = new HashSet<string>();
+
+        public void CheckIsValid()
```

Comment:
`ICommand` now has a `ValidationErrors` method that should return the `validationErrors` list.
`CheckIsValid` in the `Query` property will need to be changed to `this.CheckIsValid`, to pick up the new extension method.